prow/jobs/kyma/kyma-ondemand.yaml

```diff
@@ -0,0 +1,44 @@
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+job_template: &job_template
+  skip_report: true
+  run_if_changed: "^tools/watch-pods" #TODO only temporary solution for testing purposes
```

Comment:
As discussed, this can be replaced with sth more generic, just mark this job as not run always and triggered by a comment on GitHub. Thanks to that we will be able to execute this job on every PR just by adding a comment

Reply:
done